content/guides/code-splitting-css.md

```diff
@@ -36,13 +37,13 @@ module.exports = {
     module: {
         rules: [{
             test: /\.css$/,
-            use: 'css-loader'
+            use: [ 'style-loader', 'css-loader' ]
         }]
     }
 }
 ```
 
-As a result, the CSS is bundled along with your JavaScript.
+As a result, the CSS is bundled along with your JavaScript and applied to the page after it loaded.
```

Comment:
"...and applied to the page via '<style>' tag injection after the initial load." to elaborate a bit.